src/Shared/SFA.DAS.ProviderPayments.Calc.Shared.IntegrationTests/DbSetupScripts/6_PeriodEnd.Deds.PaymentsDue.Add_APIIndexes.sql

```diff
@@ -0,0 +1,20 @@
+IF NOT EXISTS(SELECT NULL FROM sys.indexes WHERE name='IX_RequiredPayments_AccountId')
```

Comment:
This is a direct copy from existing paymentdue integration test. Whilst I totally agree there is great value in re-working the integration test infrastructure and coming up with a good initialisation pattern, I think this should be done for all the int tests.

Reply:
It's still wrong though. It should be linked so that changes to the source are reflected by changes to the integration tests. Otherwise we're just asking for a very subtle bug down the line when we change something and the tests continue to work when they shouldn't